threescale_api/resources.py

```diff
@@ -531,10 +531,13 @@ def insert(self, index: int, *policies):
 class OIDCConfigs(DefaultClient):
     @property
     def url(self) -> str:
-        return self.parent.url + '/oidc_configuration'
+        return self.parent.url + '/proxy/oidc_configuration'
```

Comment:
this reveals that there are two implementations to access oidc. from Proxies class and from Service class. they seem to be clear duplicate. This will break proxy.oidc that is already heavily used outside.

Suggestion is either remove Service.oidc or implement it with help of Proxies.oidc

Reply:
I've solved it.